.buildkite/scripts/steps/integration_tests_tf.sh

```diff
@@ -35,8 +29,12 @@ mage build:testBinaries
 # BUILDKITE_RETRY_COUNT > 0 for the retries
 if [[ "${BUILDKITE_RETRY_COUNT}" -gt 0 ]]; then
   echo "~~~ The steps is retried, starting the ESS stack again"
+  echo "~~~ Getting stable stack version"
+  DEFAULT_STACK_VERSION="$(cat .package-version)-SNAPSHOT"
+  STABLE_ESS_VERSION="$(getStableEssSnapshotForBranch)-SNAPSHOT"
```

Comment:
ditto

Reply:
That's right. Updated the doc for the function